Makefile

```diff
@@ -15,13 +15,14 @@ tidy:
 
 GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint
 golangci-lint:
+	rm -rf bin || :
```

Comment:
Why remove entire dir, why not just remove the file e.g. rm -f $(GOLANGCI_LINT) ?

Reply:
I agree we can just remove the `rm -f $(GOLANGCI_LINT)`

Reply:
Took care of it!